hotel_housekeeping/models/hotel_housekeeping.py

```diff
@@ -0,0 +1,234 @@
+        @param self: object pointer
+        """
+        self.state = 'done'
+        if not self.quality:
+            raise ValidationError(_('Please update quality of work!'))
+
+    @api.multi
+    def room_inspect(self):
+        """
+        This method is used to change the state
+        to inspect of the hotel housekeeping
+        ---------------------------------------
+        @param self: object pointer
+        """
+        self.state = 'inspect'
+        self.quality = False
+
+    @api.multi
+    def room_clean(self):
+        """
+        This method is used to change the state
+        to clean of the hotel housekeeping
+        ---------------------------------------
+        @param self: object pointer
+        """
+        self.state = 'clean'
+        for line in self:
+            line.quality = False
+            for activity_line in line.activity_lines:
+                activity_line.clean = True,
+                activity_line.dirty = False
+
+
+class HotelHousekeepingActivities(models.Model):
+
+    _name = "hotel.housekeeping.activities"
+    _description = "Housekeeping Activities "
+
+    a_list = fields.Many2one('hotel.housekeeping', string='Reservation')
+    today_date = fields.Date('Today Date')
+    activity_name = fields.Many2one('hotel.activity',
+                                    string='Housekeeping Activity')
+    housekeeper = fields.Many2one('res.users', string='Housekeeper',
+                                  required=True)
+    clean_start_time = fields.Datetime('Clean Start Time',
+                                       required=True)
+    clean_end_time = fields.Datetime('Clean End Time', required=True)
+    dirty = fields.Boolean('Dirty',
+                           help='Checked if the housekeeping activity'
+                           'results as Dirty.')
+    clean = fields.Boolean('Clean', help='Checked if the housekeeping'
+                           'activity results as Clean.')
+
+    @api.constrains('clean_start_time', 'clean_end_time')
+    def _check_clean_start_time(self):
+        '''
+        This method is used to validate the clean_start_time and
+        clean_end_time.
+        ---------------------------------------------------------
+        @param self: object pointer
+        @return: raise warning depending on the validation
+        '''
+        if self.clean_start_time >= self.clean_end_time:
+            raise ValidationError(_(
+                'Start Date Should be less than the End Date!'))
+
+    @api.model
+    def default_get(self, fields):
+        """
+        To get default values for the object.
+        @param self: The object pointer.
+        @param fields: List of fields for which we want default values
+        @return: A dictionary which of fields with values.
+        """
+        if self._context is None:
+            self._context = {}
+        res = super(HotelHousekeepingActivities, self).default_get(fields)
```

Comment:
`super` could be called w/o arguments in Python3:
`super().default_get(fields)`